### DISCUSSION TITLE
Use gh CLI to add or modify repository description

### DISCUSSION THREAD

**Discussion by rockyhotas** [community/community#22456] [New to GitHub]
<p>Hello!<br>
I’m new to the <a href="https://cli.github.com/" rel="noopener nofollow ugc">GitHub CLI</a> <code>gh</code>. I would like to use it to add or to change the <code>description</code> of a repository.</p>
<p>The <a href="https://cli.github.com/manual/gh_repo" rel="noopener nofollow ugc">manual page</a> of <code>gh repo</code> seems not to mention this.</p>
<p>Is it possible to perform this operation with <code>gh</code>? If yes, how?</p>

**Comment by anxumalo** [comment, marked as answer]
<p>Hi <a class="mention" href="/rockyhotas">@rockyhotas</a>,</p>
<p>Yes; you have two options for editing the description in your repo:</p>
<ol>
<li>
<p><em>Using GitHub CLI:</em><br>
Try <code>gh repo edit</code> with the <code>-d</code> or <code>--description</code> flag.</p>
</li>
<li>
<p><em>Using the GitHub UI:</em><br>
In the <strong>About</strong> section of your repo, click the gear :gear:  symbol to display the <strong>Edit repository details</strong> pane. Edit the description and click <strong>Save changes</strong> once you’re done.</p>
</li>
</ol>
<p>The CLI command format is:</p>
<pre><code class="lang-auto">gh repo edit &lt;repository&gt; --description &lt;string&gt;
</code></pre>
<p>See details in the <code>manual/gh_repo_edit</code> page (I can’t add a direct URL for some reason :woman_shrugging:t3: )</p>
<p>Let me know if it works!</p>

**Reply by pratap-panabaka** [reply]
Thank You. It worked as intended.

**Reply by krystiaL** [reply]
Hi, try to use the SSH instead of the repository URL which looks like this:
```bash
    git@github.com:<your_github_username>/<repository_name>.git
```

**Comment by rockyhotas** [comment]
<p>Thank you so much!<br>
I tried your option 1 and it works! This is the related <a href="https://cli.github.com/manual/gh_repo_edit" rel="noopener nofollow ugc">manual page</a>.</p>
